cookbooks/aws-parallelcluster-platform/resources/intel_hpc/intel_hpc.rb

```diff
@@ -22,5 +22,66 @@
 end
 
 action :configure do
-  # do nothing
+  return unless intel_hpc_supported? && (node['cluster']['install_intel_base_toolkit'] == "true" || node['cluster']['install_intel_hpc_toolkit'] == "true" || node['cluster']['install_intel_python'] == "true")
+
+  base_toolkit_name = "l_BaseKit_p_2023.2.0.49397_offline.sh"
```

Comment:
The version of packages should be defined as dedicated attributes to give them more visibility and adhere to the approach we have for other packages.

Reply:
Done